### PR TITLE
Extended entity base and tests

### PR DESCRIPTION
**Pull request by roman-fm**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
     "require": {
         "php": ">=5.5.0",
         "dibi/dibi": "^3.0.6",
-        "obo/obo": ">=0.12.7",
+        "obo/obo": "dev-master",
```
Comment (hAdamm) [inline composer.json]: Is it really good? Better would be a version 0.13.0

Comment (roman-fm) [inline composer.json]: Will be added to the separate pull request
